src/data/uma-title.csv

```diff
@@ -42,7 +42,7 @@
 "[Nevertheless]","[Nevertheless]"
 "[うららん一等賞♪]","[Uraran First Prize ♪]"
 "[運気上昇☆幸福万来]","[Rising Fortunes ☆ Bountiful Blessings]"
-"[吉兆・初あらし]",""
-"[ブルー/レイジング]",""
+"[吉兆・初あらし]","[Good Omen First Storm]"
```

Comment:
The・got removed here. On the actual translation, that seems accurate enough? Gametora has it like this as well.

Reply:
It was copied from Gametora indeed. I removed the ・ for consistency as none of the other translations in the file kept it either since it functions as a space. 
This _is_ the only use of it outside of katakana though... I'll add it back to split the 2 parts.